lib/ipv6.js

```diff
@@ -409,11 +403,13 @@ Address6.prototype.getScope = function () {
 Address6.prototype.getType = function () {
   var self = this;
 
-  function isType(name, type) {
-    return self.isInSubnet(new Address6(type));
+  function isType(entry) {
+    return self.isInSubnet(new Address6(entry[0]));
   }
 
-  return find(constants6.TYPES, isType) || 'Global unicast';
+  var entry = Object.entries(constants6.TYPES).find(isType);
+
+  return entry ? entry[1] : 'Global unicast';
```

Comment:
any way to make this one more readable?

Reply:
I don't know, something like:

```js
  function isType(entry) {
    return self.isInSubnet(new Address6(entry[0]));
  }

  var [type, name] = Object.entries(constants6.TYPES).find(isType) || [];

  return name || 'Global unicast';
```

?

Reply:
Or how about:

```js
  for (const type of Object.keys(constants6.TYPES)) {
    if (self.isInSubnet(new Address6(type))) {
      return constants6.TYPES[type]
    }
  }

  return 'Global unicast';
```

Reply:
Can I get your opinion on this @beaugunderson so I can get this ready for merge?

Reply:
ping @beaugunderson